packages/sd-card-slots/Korean_Hroparts_Elec_TF_01A_model.ato

```diff
@@ -0,0 +1,18 @@
+#pragma experiment("FOR_LOOP")
+
+from "sd-card-slots.ato" import SDReceptacle
+from "parts/Korean_Hroparts_Elec_TF_01A/Korean_Hroparts_Elec_TF_01A.ato" import Korean_Hroparts_Elec_TF_01A_package
+
+module Korean_Hroparts_Elec_TF_01A_model from SDReceptacle:
+    package = new Korean_Hroparts_Elec_TF_01A_package
+
+    sdbus.CD_DAT3 ~ package.CDDAT3CS
+    sdbus.CMD ~ package.CMDDI
+    sdbus.VSS1 ~ package.VSS
+    sdbus.VDD ~ package.VDD
+    sdbus.CLK ~ package.CLKSCLK
+    sdbus.VSS2 ~ package.VSS
+    sdbus.DAT0 ~ package.DAT0D0
+    sdbus.DAT1 ~ package.DAT1RSV
+    sdbus.DAT2 ~ package.DAT2RSV
+    sdbus.CD ~ package.CD
```

Comment:
```suggestion
    sdbus.CD ~ package.CD
    sdbus.VSS1 ~ package.GND
```

I actually don't understand why sdbus has `VSS1` and `VSS2`. This should probably be `VSS` + a separate Electrical for `SHIELD` (which is Called `GND` in this package).